pkg/bytecode/vm_test.go

```diff
@@ -85,19 +86,118 @@ func TestVMArithmetic(t *testing.T) {
 			},
 		},
 		{
-			name: "addition and subtraction",
```

Comment:
removed addition and subtraction test as it is covered by the "all operators" test